src/redux/categories/categoriesSlice.js

```diff
@@ -1,11 +1,14 @@
-// En src/redux/categories/categoriesSlice.js
 import { createSlice } from '@reduxjs/toolkit';
 
+const initialState = {
+  categories: '',
+};
+
 const categoriesSlice = createSlice({
   name: 'categories',
-  initialState: 'Under construction',
+  initialState,
   reducers: {
-    checkStatus: (state) => state,
+    getCategoryStatus: () => ({ categories: 'Under Construction' }),
   },
```

Comment:
- [ ] Good job, you are almost there! :muscle: It's just a bit challenging because like the previous reviewers mentioned, clicking on the categories is not returning "Under Construction" as the requirement expected. Kindly check and fix this. Have you tested the code snippet that the immediate code reviewer suggested?